src/components/Swap/Toolbar/index.tsx

```diff
@@ -16,8 +16,10 @@ import * as Caption from './Caption'
 const ToolbarRow = styled(Row)`
   border: 1px solid ${({ theme }) => theme.outline};
   border-radius: ${({ theme }) => theme.borderRadius - 0.25}em;
+  flex-wrap: nowrap;
   gap: 0.5em;
   min-height: 3.5em;
+  overflow: hidden;
```

Comment:
In which case is it overflowing?

Reply:
when the width is really small and I add "flex-wrap", the content started overflowing. i don't think we want multi-line wrapping so i'm taking the approach of truncating the content to fit in the smaller width